### PR TITLE
Replace PerfMeter tests

### LINKED ISSUE #1255: More intermittent test failures
Build here: https://travis-ci.org/mozilla/activity-stream/builds/158822759

```
console.info: activity-streams: pass: undefined
console.info: activity-streams: pass: This test is done.
console.info: activity-streams: pass: There was a clean UI.
console.error: activity-streams: 
JPM [error]   Message: Error: The ID "action-button--activity-streams-activity-streams-link" seems already used.
  Stack:
    create@resource://gre/modules/commonjs/sdk/ui/button/view.js:120:11
setup@resource://gre/modules/commonjs/sdk/ui/button/action.js:60:5
@resource://gre/modules/commonjs/sdk/core/disposable.js:48:56
dispatch@resource://gre/modules/commonjs/method/core.js:119:12
Disposable<.initialize@resource://gre/modules/commonjs/sdk/core/disposable.js:71:17
constructor@resource://gre/modules/commonjs/sdk/core/heritage.js:146:23
_setupButton@resource://activity-streams/addon/ActivityStreams.js:641:20
ActivityStreams@resource://activity-streams/addon/ActivityStreams.js:128:3
getTestActivityStream@resource://activity-streams/test/lib/utils.js:69:17
@resource://activity-streams/test/test-PrefsProvider.js:4:12
findTests/</getNextTest@resource://gre/modules/commonjs/sdk/deprecated/unit-test-finder.js:162:25
findTests/</<.getNext@resource://gre/modules/commonjs/sdk/deprecated/unit-test-finder.js:195:32
runNextTest@resource://gre/modules/commonjs/sdk/deprecated/unit-test.js:532:14
start/this.onDone@resource://gre/modules/commonjs/sdk/deprecated/unit-test.js:563:7
done/</<@resource://gre/modules/commonjs/sdk/deprecated/unit-test.js:389:31
notify@resource://gre/modules/commonjs/sdk/timers.js:40:9
console.error: activity-streams: 
JPM [error]   Message: Error: The ID "action-button--activity-streams-activity-streams-link" seems already used.
  Stack:
    create@resource://gre/modules/commonjs/sdk/ui/button/view.js:120:11
setup@resource://gre/modules/commonjs/sdk/ui/button/action.js:60:5
@resource://gre/modules/commonjs/sdk/core/disposable.js:48:56
dispatch@resource://gre/modules/commonjs/method/core.js:119:12
Disposable<.initialize@resource://gre/modules/commonjs/sdk/core/disposable.js:71:17
constructor@resource://gre/modules/commonjs/sdk/core/heritage.js:146:23
_setupButton@resource://activity-streams/addon/ActivityStreams.js:641:20
ActivityStreams@resource://activity-streams/addon/ActivityStreams.js:128:3
getTestActivityStream@resource://activity-streams/test/lib/utils.js:69:17
@resource://activity-streams/test/test-PrefsProvider.js:4:12
findTests/</getNextTest@resource://gre/modules/commonjs/sdk/deprecated/unit-test-finder.js:162:25
findTests/</<.getNext@resource://gre/modules/commonjs/sdk/deprecated/unit-test-finder.js:195:32
runNextTest@resource://gre/modules/commonjs/sdk/deprecated/unit-test.js:532:14
start/this.onDone@resource://gre/modules/commonjs/sdk/deprecated/unit-test.js:563:7
done/</<@resource://gre/modules/commonjs/sdk/deprecated/unit-test.js:389:31
notify@resource://gre/modules/commonjs/sdk/timers.js:40:9
JPM [error] JavaScript error: resource://services-sync/policies.js, line 322: TypeError: engines[i]._tracker is undefined
console.log: activity-streams: [JavaScript Error: "Warning: setting the globalScore pref to the non-integer number NaN converted it to the integer number 0; to retain fractional precision, store non-integer numbers as strings." {file: "resource://gre/modules/Preferences.jsm" line: 139}]
console.log: activity-streams: [JavaScript Error: "TypeError: engines[i]._tracker is undefined" {file: "resource://services-sync/policies.js" line: 322}]
JPM [error] JavaScript error: resource://gre/modules/services-sync/engines/forms.js, line 234: TypeError: subject is null
console.log: activity-streams: [JavaScript Error: "TypeError: subject is null" {file: "resource://gre/modules/services-sync/engines/forms.js" line: 234}]
console.log: activity-streams: [JavaScript Error: "no element found" {file: "jar:file:///home/travis/firefox-46.0/firefox/browser/omni.ja!/chrome/browser/content/browser/aboutNetError.xhtml" line: 1}]
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
The build has been terminated
```

### PR DESCRIPTION
**Pull request by k88hudson**
We're gonna need some integration-y tests to replace what the old tests were replacing, but this is a start – still a couple of things to figure out 👍 

Fixes #1255

### CODE REVIEW

File: content-test/addon/PerfMeter.test.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+  let simplePrefs;
+
+  function setup(prefs = {"performance.log": true}) {
+    tabs = new Tabs();
+    simplePrefs = new SimplePrefs(prefs);
+    const {PerfMeter} = injector({"sdk/tabs": tabs, "sdk/simple-prefs": simplePrefs});
+    perfMeter = new PerfMeter([TEST_URL]);
+  }
+
+  beforeEach(() => setup());
```
Comment (ncloudioj) [inline content-test/addon/PerfMeter.test.js]: beforeEach(setup)?

File: content-test/index.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ describe("ActivtyStreams", () => {
   let restore;
   let originalAlert;
   let originalConfirm;
+  let originalServices;
```
Comment (ncloudioj) [inline content-test/index.js]: Leave it as `undefined`? Couldn't find the initialization/assignment of `originalService`.

**Comment by ncloudioj** [comment]
Looks great! :shipit:

File: content-test/index.js (hunk 2 above)
Comment (k88hudson) [inline content-test/index.js]: oops, good catch

**Pull request merged**